tests/benchmarks/search-expire-doc-1000-seconds.yml

```diff
@@ -0,0 +1,37 @@
+version: 0.2
+name: "search-expire-doc-1000-seconds"
+description: "
+             We want to check our ttl table performance, edge cases we need to consider when modifying expiration tests:
+             - we want to keep an existing dataset, if we actually expire documents during the test we will decrease our keyspace size
+             - if we reach a point where every single key is set to be expired then there will be a hot path which might not represent real world usage 
+             "
+
+metadata:
+  component: "search"
+setups:
+  - oss-standalone
+  - oss-cluster-02-primaries
+  - oss-cluster-04-primaries
+  - oss-cluster-08-primaries
+  - oss-cluster-16-primaries
+  - oss-cluster-20-primaries
+  - oss-cluster-24-primaries
+  - oss-cluster-32-primaries
+
+dbconfig:
+  - tool: ftsb_redisearch
+  - parameters:
+    - workers: 64
+    - reporting-period: 1s
+    - requests: 100000
+    - input: "https://s3.amazonaws.com/benchmarks.redislabs/redisearch/datasets/1M-nyc_taxis-hashes/1M-nyc_taxis-hashes.redisearch.commands.ALL.csv"
+  - init_commands:
+      - '"FT.CREATE" "nyc" "SCHEMA" "total_amount" "NUMERIC" "SORTABLE" "improvement_surcharge" "NUMERIC" "SORTABLE" "fare_amount" "NUMERIC" "SORTABLE" "trip_distance" "NUMERIC" "SORTABLE"'
+  - dataset_load_timeout_secs: 180
+  - check:
+      keyspacelen: 100000
+
+clientconfig:
+  benchmark_type: "read-only"
+  tool: memtier_benchmark
+  arguments: "--test-time 120 -c 32 -t 1 --hide-histogram --key-prefix '' --key-minimum 1 --key-maximum 100000 --command 'FT.SEARCH nyc @trip_distance:[6,8] LIMIT 0 0' --command-ratio 90 --command 'EXPIRE __key__  10' --command-ratio 10"
```

Comment:
@kei-nan We expire 10% of the keys - how can we know it is expired during a query?

Reply:
I can maybe disable active expiration in the setup for the benchmarks.
But I think our ttl table should be able to handle the expiration mid query, no?

Reply:
Yes, we should be able to handle mid-query lazy expirations - I suggest to make sure it is enabled and tested in this benchmark, so I agree disabling active expiration looks good.

Reply:
I meant we should be able to handle mid query active expiration, no?

Reply:
In the end we added tests that should answer this requirement.
Marking as resolved.